tests/operator_test.go

```diff
@@ -970,7 +970,7 @@ spec:
 		// running a VM/VMI using that previous release
 		// Updating KubeVirt to the target tested code
 		// Ensuring VM/VMI is still operational after the update from previous release.
-		It("[release-blocker][owner:@sig-compute][test_id:3145]from previous release to target tested release", func() {
+		It("[QUARANTINE][release-blocker][owner:@sig-compute][test_id:3145]from previous release to target tested release", func() {
```

Comment:
I think we merged something yesterday which takes out the most flaky part of the test.

Reply:
Ah no, that was for another test: https://github.com/kubevirt/kubevirt/pull/5229/files#diff-e3e6528c8b5d4860f4fd89d38f0e0babb9cc0e9147c0b0b28be532e5e692c182R1253